examples/todo-jwt/src/index.ts

```diff
@@ -3,8 +3,9 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {TodoListApplication} from './application';
-import {ApplicationConfig} from '@loopback/core';
+import {ApplicationConfig, TodoListApplication} from './application';
+
+export {TodoListApplication};
```

Comment:
In order to address comment: https://github.com/strongloop/loopback-next/pull/6590#discussion_r510173442, I had to put

```
export {TodoListApplication};
```
back.

## Background

This is all because
I removed

```
// re-exports for our benchmark, not needed for the tutorial itself
export {TodoListApplication};

export * from './models';
export * from './repositories';
export * from '@loopback/rest';

```
since there is no benchmark that uses `todo-jwt`. So all these exports aren't necessary. They were copied over inadvertently from `todo` which `is` accessed by the benchmark.

These exports don't show up in the index.ts that is generated by CLI. That is why I sought to remove them.